kolibri/core/assets/src/composables/useUserSyncStatus.js

```diff
@@ -44,23 +44,21 @@ export function pollUserSyncStatusTask() {
       deviceStatusSentiment.value = syncData[0].device_status_sentiment;
     }
   });
-
-  return status;
 }
 
 export function useUserSyncStatus() {
+  const { pause, resume } = useTimeoutPoll(pollUserSyncStatusTask, 1000);
```

Comment:
Since the call to `useTimeoutPoll` is inside this function, every time `useUserSyncStatus` is invoked it's going to create a new timer, which will not play nice with the usage tracking.

Here's a situation where this could go wrong:
1) Component A calls `useUserSyncStatus`, creates timer A
2) Since Component A is the first usage of `useUserSyncStatus`, it will call `resume()` for timer A
3) Component B calls `useUserSyncStatus`, creates timer B
4) Since Component B is the second usage, it does not call `resume()` for timer B
5) Component A is destroyed
6) Since there is still one usage by Component B (still mounted), `pause()` for either timer won't be called after Component A is destroyed 
7) Component B is destroyed
8) Since Component B was the last usage of `useUserSyncStatus`, it will call `pause()`... BUT that `pause` function refers to the timer B not timer A. Timer A is the active timer (the one we called `resume()`), and timer B never had `resume()` called, meaning timer A will continue forever.

Since the goal is to de-duplicate the polling if more than one component uses `useUserSyncStatus` and store that polling result for any usage of `useUserSyncStatus`, we only need one timer. That means calling `useTimeoutPoll(pollUserSyncStatusTask, 1000);` should exist outside of the composable function `useUserSyncStatus`. This is what I meant in my previous comment about it being at the same scope as the refs.